src/main/java/org/molgenis/vcf/inheritance/matcher/AppCommandLineOptions.java

```diff
@@ -54,6 +56,12 @@ class AppCommandLineOptions {
             .longOpt(OPT_PROBANDS_LONG)
             .desc("Comma-separated list of proband sample identifiers.")
             .build());
+    appOptions.addOption(
+            Option.builder(OPT_CLASSES)
```

Comment:
how will this work when using this tool in VIP? suppose I create my own classification tree with leaf nodes `keep` and `remove`?

Reply:
Either leave it empty, assuming only "keep" reaches the inheritance matcher, or specify "keep" in this option, to prevent "remove" variants from being processed (and ending up as part of a compound)

Reply:
the description has been updated, since it was outdated and referred to a mapping that is no longer there in the current solution